Bringing more exposure to Sample App (#56)
wwt/SwiftCurrent: README.md

As you continue to develop your applications, each view controller will become more decoupled from the rest of the app.  That means, if you want a completely different order of screens, just define a new [Workflow](https://wwt.github.io/SwiftCurrent/Classes/Workflow.html).

## See it in action with our sample app

> Clone our repo, open `SwiftCurrent.xcworkspace`, target the `SwiftCurrentExample` scheme, and run to see our sample app in action. The app is designed to give you an idea of what SwiftCurrent can do with minimal overhead in the UI.

Reviewer: Make names for the sample app and example app. The lack of differentiation could lead to confusion.

Author: Thanks for the call out.  I'm just going to consolidate to example app for now.

## Interested but you need SwiftUI support?

[We're working on it now!](https://github.com/wwt/SwiftCurrent/milestone/2)

# Quick Start

> This quick start uses SPM, but if you would prefer CocoaPods, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).

Reviewer 2: "but if you would prefer CocoaPods". Is this really what we want to say? In the spirit of keeping documentation from getting out of date, "But if you're interested in using a different installation method....".

Author: How about?

```suggestion
This quick start uses SPM, but for a complete list of supported package managers, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).
```

Reviewer 2: It's acceptable, but the really pedantic side of me wants to say that our installation instructions *could* include instructions without a package manager. I certainly strongly recommend that people consume our library through a package manager of some kind, but not everyone will choose to do that.

Reviewer 3: Something I tossed around looking at this spot before was

```suggestion
This quick start uses SPM, but for other dependency approaches, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).
```

Author: Just dropping in Tyler's suggestion
```suggestion
This quick start uses SPM, but if you're interested in using a different installation method, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).
```

Author: I think I like Morgan's the best so far.  Maybe:

```suggestion
This quick start uses SPM, but for other approaches, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).
```

launchInto(Workflow(ExampleViewController.self))
```

> ### Original:

Author: There's always option 3, which is to not even call out the example app or the getting started.  And just end this section.

Author: This option was the winner

And just like that you're started!  To see something more practical and in-depth, check out the example app in the repo.  For a more in-depth starting guide, checkout out our [Getting Started](https://github.com/wwt/SwiftCurrent/wiki/getting-started) documentation.

> ### Option #1:

Author: I like that this one offers up clean links to the guides and links back to the example app instructions, but it seems a bit much.

And just like that you're started!  If you would like to see a more practical example, [checkout the example app in the repo]().  If you would like to have a slightly larger example than this quick start that may answer a number of initial questions you might have, take a look at the getting started guide that is most relevant to you:

- [Getting Started with Storyboards](https://github.com/wwt/SwiftCurrent/wiki/getting-started)
- [Getting Started with Programmatic UIKit](https://github.com/wwt/SwiftCurrent/wiki/Getting-Started-with-Programmatic-UIKit)

> ### Option #2:

Author: I think I like this option better, but it still seems a bit weird.  Maybe it's good enough for now.  But it will definitely get out dated when SwiftUI comes along.

> And just like that you're started!  To see something more practical and in-depth, check out the example app in the repo.  For a more in-depth starting guide, checkout out our [Getting Started](https://github.com/wwt/SwiftCurrent/wiki/getting-started) documentation.

Reviewer: It is a practical example, as we believe this would best suit consumer's needs. The Getting Started app includes detailed explanations for a trivial. 
 
 For a quick start, we've included an example app that includes detailed comments and explanations.

Author: Thanks for spotting this section.  I completely forgot to update it when I added the new call-out at the top.
